src/serverconf/src/main/java/ee/ria/xroad/common/conf/serverconf/IsAuthentication.java

```diff
@@ -89,11 +92,13 @@ public static void verifyClientAuthentication(ClientId client,
                 throw new CodedException(X_SSL_AUTH_FAILED,
                         "Client (%s) has no IS certificates", client);
             }
-
-            if (!isCerts.contains(auth.getCert())) {
-                throw new CodedException(X_SSL_AUTH_FAILED,
-                        "Client (%s) TLS certificate does not match any"
-                                + " IS certificates", client);
```

Comment:
The "does the cert exist" check is missing.

Reply:
Should be back now